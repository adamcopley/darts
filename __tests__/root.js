import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import Root from '../app/root'

test('Root renders correctly', () => {
    let tree = renderer.create(
        <Root />
    ).toJSON()

    expect(tree).toMatchSnapshot()
})