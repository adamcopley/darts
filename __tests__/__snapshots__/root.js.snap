// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "fontSize": 40,
        "marginBottom": 20,
      }
    }
  >
    p1
     vs 
    p2
  </Text>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "black",
          "fontSize": 30,
          "marginTop": 10,
        }
      }
    >
      Player 1:
    </Text>
    <TextInput
      onChangeText={[Function]}
      placeholder="p1"
      style={
        Object {
          "height": 25,
        }
      }
    />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "black",
          "fontSize": 30,
          "marginTop": 10,
        }
      }
    >
      Player 2:
    </Text>
    <TextInput
      onChangeText={[Function]}
      placeholder="p2"
      style={
        Object {
          "height": 25,
        }
      }
    />
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#F5FCFF",
          "flex": 1,
          "justifyContent": "center",
        },
        Object {
          "flex": 0,
        },
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "black",
          "fontSize": 30,
          "marginTop": 10,
        }
      }
    >
      Total Score:
    </Text>
    <View
      style={
        Object {
          "width": 90,
        }
      }
    >
      <RCTPicker
        items={
          Array [
            Object {
              "label": "101",
              "textColor": undefined,
              "value": "101",
            },
            Object {
              "label": "301",
              "textColor": undefined,
              "value": "301",
            },
            Object {
              "label": "501",
              "textColor": undefined,
              "value": "501",
            },
          ]
        }
        onChange={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        selectedIndex={1}
        style={
          Array [
            Object {
              "height": 216,
            },
            undefined,
          ]
        }
      />
    </View>
  </View>
  <View
    accessibilityComponentType="button"
    accessibilityLabel={undefined}
    accessibilityTraits={
      Array [
        "button",
      ]
    }
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#0C42FD",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Start game
      </Text>
    </View>
  </View>
</View>
`;
